src/app/GitUI/LeftPanel/RepoObjectsTree.cs

```diff
@@ -471,7 +471,7 @@ private void DoSearch()
     {
         _txtBranchCriterion.CloseDropdown();
 
-        if (_searchCriteriaChanged && _searchResult?.Any() is true)
+        if (_searchCriteriaChanged && _searchResult is not null && _searchResult.Count > 0)
```

Comment:
```suggestion
        if (_searchCriteriaChanged && _searchResult.?Count is > 0)
```

Reply:
(it is of course if (_searchCriteriaChanged && _searchResult?.Count is > 0) - not meaning to nitpicking)